client/state/account-recovery/test/actions.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { useNock } from 'test/helpers/use-nock';
+
+import {
+	accountRecoveryFetch,
+	accountRecoveryFetchSuccess,
+	accountRecoveryFetchFailed,
+} from '../actions';
+
+import {
+	ACCOUNT_RECOVERY_FETCH,
+	ACCOUNT_RECOVERY_FETCH_SUCCESS,
+	ACCOUNT_RECOVERY_FETCH_FAILED,
+} from 'state/action-types';
+
+import dummyData from './test-data';
+
+describe( 'account-recovery ctions', () => {
+	const spy = sinon.spy();
+
+	beforeEach( () => {
+		spy.reset();
```

Comment:
If you use the [`use-sinon` test helper](https://github.com/Automattic/wp-calypso/tree/master/test/test/helpers/use-sinon), you don't need to reset yourself:

```js
let spy;
useSandbox( ( sandbox ) => spy = sandbox.spy() );
```